RFC/src/RFC-0171_MessageSerialisation.md

```diff
@@ -135,52 +135,54 @@ Several candidates fulfil these properties to some degree.
   * Very compact
   * Fast
   * Multiple language support
-  * Good Rust / Serde support
-  * Dovetails into ZMQ nicely
+  * Good Rust/Serde support
+  * Dovetails nicely into ZMQ 
 * Cons:
   * No metadata support
 
 #### [Protobuf](https://code.google.com/p/protobuf/)
 
-Similar to [Message Pack](#message-pack), but also requires schema's to be written and compiled. Serialisation performance and size
-is similar to Message Pack. Can work with ZMQ but is better designed to be used with gRPC.
+Similar to [Message Pack](#message-pack), but also requires schemas to be written and compiled. Serialization 
```

Comment:
```suggestion
Similar to Message Pack, but also requires schemas to be written and compiled. Serialization 
```

Reply:
The internal link to _Message Pack_ does not work because _Message Pack_ in its turn has an external link to it.